resources/compass/charts/cockpit/OWNERS

```diff
@@ -3,5 +3,8 @@ reviewers:
 
 approvers:
   - kwiatekus
+  - dariadomagala
   - y-kkamil
-  - sjanota 
```

Comment:
Please remove y-kkamil

Reply:
Ok, but please let me merge this one and quickly open a new PR for that, I would like to avoid another round of retest roulette.